docs/tutorials/elasticdl_local.md

```diff
@@ -99,22 +99,51 @@ retrieve the source code, please run the following command.
 git clone https://github.com/sql-machine-learning/elasticdl
 ```
 
-The model definitions are in directory `elasticdl/model_zoo`.  The following
-commands build the Docker image `elasticdl:mnist`
+The model definitions are in directory `elasticdl/model_zoo`.
+
+### Build the Model Zoo for Parameter Server
+
+The following commands build the Docker image `elasticdl:mnist_ps`
+
+```bash
+cd elasticdl
+elasticdl zoo init --model_zoo=model_zoo
+elasticdl zoo build --image=elasticdl:mnist_ps .
+```
+
+### Build the Model Zoo for AllReduce
+
```

Comment:
We have not released ElasticDL packages with AllReduce yet. Thus, we need to manually build packages with AllReduce support.

We must build an image `elasticdl:dev_allreduce` firstly using the

```bash
scripts/travis/build_images.sh
```

Then we use this image to build packages with AllReduce support.

```bash
scripts/docker_build_wheel.sh
```

After this, we can build the AllReduce training image `elasticdl:mnist_allreduce`
with model definitions in `model_zoo`.